src/Modules/ExteriorPowers/FreeModules.jl

```diff
@@ -9,42 +9,44 @@
 
 # User facing constructor for ⋀ ᵖ F.
 function exterior_power(F::FreeMod, p::Int; cached::Bool=true)
-  (p < 0 || p > rank(F)) && error("index out of bounds")
+  @req 0 <= p <= rank(F) "Exponent out of bounds"
```

Comment:
```suggestion
  @req 0 <= p <= rank(F) "exponent out of bounds"
```
This should be the convention: Don't capitalize.